sdk/core/core-crypto/rollup.base.config.js

```diff
@@ -0,0 +1,118 @@
+import path from "path";
```

Comment:
can you use the common rollup config that @witemple-msft set up inside of devtool? Look at Text Analytics for an example

Reply:
I updated the browser/nodejs builds to use the common rollup from devtool. I have a separate rollup config for creating the webworker script that runs in our tests.